.eslintrc.json

```diff
@@ -6,6 +6,7 @@
     "plugin:react/recommended",
     "plugin:react/jsx-runtime",
     "plugin:react-hooks/recommended",
+    "plugin:jest-dom/recommended",
```

Comment:
I think there is a way to add eslint only for specific files (test files in this case). Can you add it besides `plugin:jest/recommended` that you can see in `overrides` ?

Reply:
done